heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -479,7 +479,7 @@ contains JSON:
       'X-API-Key': '12345-mykey-67890'
   check.response:
     status: 200
-    body: '{"status": "ok"}'
+    body: {"status": "ok"}
```

Comment:
The input is yaml. By using single quotes, the body will be parsed as a string. Single quotes will be removed, double quotes will be kept. By removing the single quotes, `body` is parsed as an object (with double quotes being removed). Checking the configuration object in heartbeat (has been changed recently), body must be a list of regular expressions:

```
body: ['{"status":\s*"ok"}']
```

Using `\s+` we make whitepsace optional.